package.json

```diff
@@ -30,7 +30,7 @@
     "gatsby-plugin-manifest": "^2.2.30",
     "gatsby-plugin-mdx": "^1.2.27",
     "gatsby-plugin-react-helmet": "^3.1.16",
-    "gatsby-plugin-sharp": "^2.3.4",
+    "gatsby-plugin-sharp": "^2.6.36",

```

Comment:
Please revert

Reply:
It it ok to move forward? Or does moving forward cause an unwanted breakage?

Reply:
It did show some warnings for me on the older version, and I ran `npm audit fix` and bumped up the dependencies.

Reply:
Ok, unless @kushthedude comes back to advise otherwise, let's push forward with the higher version number.